types/expo/expo-tests.tsx

```diff
@@ -609,7 +609,7 @@ const allSecureStoreKeychainAccessibleValues: number[] = [
         </Svg.G>
         <Svg.Use href="#shape" transform="translate(0, 0)" x="20" y="0" />
         <Svg.Use href="#shape" transform={{ translateX: 0, translateY: 0 }} x="20" y="0" width="20" height="20"/>
-        <Svg.Symbol id="symbol" viewBox="0 0 150 110" width="100" height="50">
```

Comment:
not sure about this one. `symbol` seems to support `width` and `height`: https://developer.mozilla.org/en-US/docs/Web/SVG/Element/symbol

/cc @msand

Reply:
@SimenB It seems the geometry properties x, y, width and height were added in Svg 2.0: https://svgwg.org/svg2-draft/struct.html#SymbolElement
react-native-svg doesn't have support for it yet at least, might be better to wait until it's implemented before adding it to the types. As otherwise people might get surprised when the types suggest it's available, but it has no visible impact. And, if it gets implemented later on, then they would likely get a different output once they upgrade.

Reply:
That makes sense, thanks!

Reply:
@msand: So SimonB did the right thing, no? And so we should give this a thumbs up, correct? Or did I miss something?

Reply:
Yeah, this is resolved, I just kept the conversation around instead of hitting "resolve" so reviewers could see this